### PR TITLE
add donation page

### PR DESCRIPTION
**Pull request by SSgumS**
some small fixes

### CODE REVIEW

File: guides/installation.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Installation
 
-In order to install Pyegi, you may clone it from its [Github page](https://github.com/SSgumS/Pyegi), extract the zipped file and using windows PowerShell, run the file "install.bat" from the extracted files. Right now, the package is available for Windows operating systems.
+In order to install Pyegi, you may clone it from its [Github page](https://github.com/SSgumS/Pyegi), extract the zipped file and using windows PowerShell, run the file `install.bat` from the extracted files. Although the installation process is currently only available for Windows, the Pyegi itself is cross-platform compatible, so if it is installed correctly, it should run on other platforms as well.
```
Comment (DrunkSimurgh) [inline guides/installation.md]: that "installed correctly" is a big if. they need to know the whole process of installation in Windows and replace the steps they need for their OS

File: how-to-use/for-developers/write-script/using-pyegi-and-pyonfx-api.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Currently, in order to write a python script for subtitle editing, you need to i
 import Pyegi
 ```
 
-We currently support [PyonFX](https://github.com/CoffeeStraw/PyonFX) library which helps with subtitle editing. In order to receive the lines from Aegisub, you need to include this code:
+You can add [PyonFX](https://github.com/CoffeeStraw/PyonFX) library to dependencies of the `pyproject.toml` file which helps with subtitle editing. In order to receive the lines from Aegisub, you need to include this code:
```
Comment (DrunkSimurgh) [inline how-to-use/for-developers/write-script/using-pyegi-and-pyonfx-api.md]: this is a little ambiguous. developers may take this as a suggestion and go with their library of choice which is not necessarily PyonFX

File: guides/installation.md (hunk 1 above)
Comment (SSgumS) [reply guides/installation.md]: The point is that Pyegi is cross-platform by nature. Obviously, `the installation process is currently only available for Windows`.

File: how-to-use/for-developers/write-script/using-pyegi-and-pyonfx-api.md (hunk 2 above)
Comment (SSgumS) [reply how-to-use/for-developers/write-script/using-pyegi-and-pyonfx-api.md]: fixed: https://github.com/DrunkSimurgh/Pyegi_Docs/pull/1/files#diff-48a6f7b5121997825ec46018587b982ac0d2393eec0872f9868c0329e1844a1dR9

File: guides/installation.md (hunk 1 above)
Comment (DrunkSimurgh) [reply guides/installation.md]: still...
i would go with "if installed manually" or sth like this

Comment (SSgumS) [reply guides/installation.md]: fixed

**Pull request merged**